CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/PluginParametersSwingDialog.java

```diff
@@ -143,26 +151,79 @@ public void setSize(final Dimension dimension) {
         xp.setPreferredSize(dimension);
     }
 
+    /**
+     * Generates a DialogDescripter window and waits until the user
+     * to select an option (the acceptance option will be highlighted). 
+     * The acceptance option is "OK" by default.
+     * The rejectionoOption is "Cancel" by default. 
+     * If a keyword is present in the title of the DialogDisplayer
+     * the acceptance option will adjust dynamically to express that keyword 
+     * (i.e. "Build", "Save", "Export")
+     */
     public void showAndWait() {
-        final DialogDescriptor dd = new DialogDescriptor(xp, title);
+        final DialogDescriptor dd = createDialogDescriptor(true);  
         final Object r = DialogDisplayer.getDefault().notify(dd);
-        if (r == DialogDescriptor.OK_OPTION) {
+        if (r == DialogDescriptor.CANCEL_OPTION){
+            result = CANCEL;
+        } else if (r == DialogDescriptor.OK_OPTION){
             result = OK;
         } else {
-            result = r == DialogDescriptor.CANCEL_OPTION ? CANCEL : null;
+            result = null;
         }
     }
 
+    /**
+     * Generates a DialogDescripter window and waits until the user
+     * to select an option (No option will be highlighted). 
+     * The acceptance option is "OK" by default.
+     * The rejectionoOption is "Cancel" by default. 
+     * If a keyword is present in the title of the DialogDisplayer
+     * the acceptance option will adjust dynamically to express that keyword 
+     * (i.e. "Build", "Save", "Export")
+     */
     public void showAndWaitNoFocus() {
-        //Having 'No' button as initial value means focus is off of 'OK' and 'Cancel' buttons
-        final DialogDescriptor dd = new DialogDescriptor(xp, title, true, DialogDescriptor.OK_CANCEL_OPTION, DialogDescriptor.NO_OPTION, null);
+        final DialogDescriptor dd = createDialogDescriptor(true);  
```

Comment:
it seems this is meant to pass false instead rather than true

Reply:
Good catch, thanks :) Copy and pasting oversight on my part when I was working on the code below